test/e2e/placement_selecting_resources_test.go

```diff
@@ -1185,6 +1186,145 @@ var _ = Describe("validating CRP when selected resources cross the 1MB limit", O
 	})
 })
 
+// This test verifies that resources are selected and ordered correctly according to the sortResources logic
+var _ = Describe("creating CRP and checking selected resources order", Ordered, func() {
+	crpName := fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())
+	nsName := fmt.Sprintf("test-ns-order-%d", GinkgoParallelProcess())
```

Comment:
Hi Ryan! Just a nit: the NS part could use the shared `appNamespace()` utility/`workNamespaceNameTemplate` variable/`workResourceSelector()` utility to simplify some of the setup.

Reply:
actully I found that e2e has to use the appNamespace() as the clean up assumed that. Bummer.